megamek/src/megamek/common/Terrains.java

```diff
@@ -211,38 +211,16 @@ public static String getEditorTooltip(int type) {
     }
 
     /**
+     * Returns a name to be displayed in tooltips. 
+     * Intentionally returns null for terrains that should be treated
+     * separately such as buildings and for terrains that should not be listed
+     * such as cliff-tops.
      * @param type the type of terrain to get the name for
      * @param level the level of the terrain to get the specific name
      * @return a displayable name for this terrain (for tooltips)
      */
     public static String getDisplayName(int type, int level) {
         switch (type) {
-        case (BUILDING):
-            if (level == 1) {
-                return "Light building";
-            } else if (level == 2) {
-                return "Medium building";
-            } else if (level == 3) {
-                return "Heavy building";
-            } else if (level == 4) {
-                return "Hardened Building";
```

Comment:
I assume that this descriptive building text is contained somewhere else?

Reply:
No, this was not a good change. I'll revert it.